packages/CodingStandard/tests/PHPUnit/Constraint/FixedSameStringsConstraint.php

```diff
@@ -0,0 +1,42 @@
+<?php declare(strict_types=1);
```

Comment:
so sad to violate the license again :(

Reply:
It just hack, but feel free to add it

Reply:
that's the thing, if you copy-paste the code, you need to respect the license, not wait for guys to realise you did it....